src/ng-xform/ng-xform.component.ts

```diff
@@ -88,7 +81,7 @@ export class NgXformComponent implements OnInit, OnChanges {
     let modelToSend = { ...this.model };
     for (const attr in this.form.value) {
       if (this.form.value.hasOwnProperty(attr)) {
-        modelToSend[attr] = this.getAttributeValue(attr, this.form.value[attr]);
+        modelToSend[attr] = this.form.value[attr] || null;
```

Comment:
Can we use `Object.assign()`? Would be a lot less code

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign

Reply:
no, because we need to overwrite empty strings to null.